src/engraving/libmscore/tie.cpp

```diff
@@ -98,12 +98,12 @@ bool TieSegment::edit(EditData& ed)
 {
     SlurTie* sl = tie();
 
-    if (ed.key == Qt::Key_X) {
+    if (ed.key == Qt::Key_X && !ed.modifiers) {
```

Comment:
TBH I don't like this, I'd rather this be in response to the "flip placement" command which is normally mapped to "X", but this is a minimal fix.
Note that MU3 allows using, e.g. CTRL+ALT+X to flip a slur if CTRL+ALT+X isn't mapped to another command, but this isn't really expected/documented behaviour.